tests/containers/podman_integration.pm

```diff
@@ -12,19 +12,46 @@ use testapi;
 use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
 use containers::utils qw(get_podman_version);
 use utils qw(zypper_call script_retry);
-use version_utils qw(get_os_release);
+use version_utils qw(get_os_release is_transactional is_sle is_sle_micro);
+use transactional qw(trup_call check_reboot_changes);
+use registration qw(add_suseconnect_product get_addon_fullname);
 use containers::common;
 
 sub run {
     select_serial_terminal;
     my ($running_version, $sp, $host_distri) = get_os_release;
     install_podman_when_needed($host_distri);
 
+    if (is_sle_micro) {
```

Comment:
shouldn't this part go to `add_suseconnect_product`?

Reply:
> shouldn't this part go to `add_suseconnect_product`?

I think that would be way outside the scope of this PR, since we're only adding PackageHub here.  Add support for SLE Micro in `add_suseconnect_product()` ss better left to another PR that handles all products.

Reply:
alright, I am just raising this point as we will forget about it otherwise